src/components/parameters/common/limitreductions/limit-reductions-table-form.tsx

```diff
@@ -15,31 +15,43 @@ import {
 } from './columns-definitions';
 import { CustomVoltageLevelTable } from '../voltage-level-table';
 
-const getLabelColumn = (limit: ITemporaryLimitReduction) => {
-    const lowBound = `${Math.trunc(limit.limitDuration.lowBound / 60)} min`;
-    const highBoundValue = Math.trunc(limit.limitDuration.highBound / 60);
-    const highBound = highBoundValue === 0 ? '∞' : `${Math.trunc(limit.limitDuration.highBound / 60)} min`;
-    const lowerBoundClosed = limit.limitDuration.lowClosed ? '[' : ']';
-    const highBoundClosed = limit.limitDuration.highClosed || null ? ']' : '[';
-    return `${lowerBoundClosed}${lowBound}, ${highBound}${highBoundClosed}`;
-};
-
 export function LimitReductionsTableForm({ limits }: Readonly<{ limits: ILimitReductionsByVoltageLevel[] }>) {
     const intl = useIntl();
 
-    const getToolTipColumn = useCallback(
+    const getLabelColumn = useCallback(
```

Comment:
We don't need to memoize those functions be cause it depends of any props or states in the `LimitReductionsTableForm` component. That's why `getLabelColumn` was in the global scope and not exported and I think we should move `getToolTipColumn` in global scope as well.
You need translation then make those functions return a <FormattedMessage/> React node type, it should work